internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -873,6 +874,27 @@ func generateBackupJobSpecIntent(ctx context.Context, postgresCluster *v1beta1.P
 		// to read certificate files
 		jobSpec.Template.Spec.SecurityContext = postgres.PodSecurityContext(postgresCluster)
 		pgbackrest.AddConfigToCloudBackupJob(postgresCluster, &jobSpec.Template)
+
+		// If the user has specified a PVC to use as a log volume via the PGBackRestCloudLogVolume
+		// annotation, check for the PVC. If we find it, mount it to the backup job.
+		// Otherwise, create a warning event.
+		if logVolumeName := postgresCluster.Annotations[naming.PGBackRestCloudLogVolume]; logVolumeName != "" {
+			logVolume := &corev1.PersistentVolumeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      logVolumeName,
+					Namespace: postgresCluster.GetNamespace(),
+				},
+			}
+			err := errors.WithStack(r.Client.Get(ctx,
```

Comment:
What wold happen if we didn't check for the PVC, but just assumed it existed and it didn't? The job pod would get stuck at pending, right?

Reply:
In this spot it would definitely break the job because it would attempt to mount a volume whose source PVC could not be found.

Reply:
OK, after reading all your comments, I understand the "backups are more important than logging" reasoning, and I'm +1 with that now.